lib/surrealist/vars_helper.rb

```diff
@@ -41,18 +43,24 @@ def set_schema(klass, hash)
       # Checks if there is a serializer defined for a given class and returns it
       #
       # @param [Class] klass a class to check
+      # @param [Symbol] tag a tag associated with serializer
       #
       # @return [Class | nil]
-      def find_serializer(klass)
-        klass.instance_variable_get(SERIALIZER_CLASS)
+      def find_serializer(klass, tag: nil)
+        tag ||= DEFAULT_TAG
+        klass.instance_variable_get(SERIALIZER_CLASSES).try(:[], tag.to_sym)
```

Comment:
We don't have ActiveSupport in here 🙃

Reply:
You can use `return` + `fetch(tag.to_sym, nil)` here I guess (safe navigation would be better, but it's not available in ruby 2.2)